VERSION_MANAGERS.md

```diff
@@ -26,14 +26,14 @@ command that will activate the right Ruby version or add the Ruby `bin` folder t
 }
 ```
 
-### rtx
+### mise (formerly rtx)
 
-[rtx](https://github.com/jdxcode/rtx) is a Rust clone compatible with asdf. You can use it by adding the following
+[mise](https://github.com/jdxcode/mise) is a Rust clone compatible with asdf. You can use it by adding the following
```

Comment:
```suggestion
[mise](https://github.com/jdx/mise) (formerly `rtx`) is a Rust clone compatible with asdf. You can use it by adding the following
```

Reply:
(once the new name is more established we can remove the `rtx` mention, but for now I think it's helpful for searching).

Reply:
👍 I've pushed an update to remove the extra mention.